pelican-gfm/gfm.py

```diff
@@ -29,8 +29,16 @@
 import pelican.utils
 import pelican.signals
 import pelican.readers
-from . import gfmSetup
-from . import Settings
+
+try:
```

Comment:
What is the purpose of this block with the relative import? If you are trying to maintain some compatibility with ancient Python versions can you solve this problem with the [PEP 328](https://www.python.org/dev/peps/pep-0328/) `from __future__ import absolute_import` approach?

Reply:
I got the absolute imports working but now the plugin doesn't conform to your documentation.